### ISSUE TITLE
Occasional tile issues after winning a Telefang

### ISSUE THREAD

**Issue by Blaziken257** [telefang/telefang#378]
Occasionally, when defeating a Denjuu in a Telefang, some glitchy tiles will appear above your Denjuu's name for a frame. It doesn't happen all the time, so I don't know how to reproduce it consistently. Nonetheless, a save state and an animation is attached below (with a single frame picked out to see it more easily).

![Telefang-1570057406-00056](https://user-images.githubusercontent.com/2187686/66088172-14ff2580-e52f-11e9-937a-1e1de18857fa.png) ![Telefang_battle_tile_glitch](https://user-images.githubusercontent.com/2187686/66088217-3102c700-e52f-11e9-81f2-ec303f6f38e4.gif)
[telefang_sp_english_battle_tile_glitch_sgb_save_state.zip](https://github.com/telefang/telefang/files/3683756/telefang_sp_english_battle_tile_glitch_sgb_save_state.zip)

**Comment by obskyr** [comment]
Those are the status effect tiles – it seems that something else is loaded into those tiles for a single frame in some circumstance. Setting a write breakpoint on that spot in VRAM may help pinpoint where that's being loaded in.

**Comment by Blaziken257** [comment]
I took a brief look at this and when I set a breakpoint at 0:0CCD in BGB, this hits when the glitchy tiles are about to be loaded in VRAM. This calls YetAnotherWFB. This seems to write a byte from de to hl. The first time this trips, de is 6FE0 and hl is 9000. The glitchy tiles are from 16-1A though, and this breakpoint trips repeatedly. By the time that hl is 9160 (the starting address of the glitchy tiles), de is 70BB. The ROM bank is 36 so it looks like the compressed tiles start at 0xDA0BB.

I haven't figured out why this gets called in the first place or where the 6FE0 value is coming from. I'll look further later on...
